photonpump/conversations.py

```diff
@@ -420,6 +479,80 @@ def _fetch_page_message(self, from_event):
     async def start(self, output):
         await output.put(self._fetch_page_message(self.from_event))
 
+class ReadAllEvents(ReadAllEventsBehaviour, PageAllEventsBehaviour):
+    """Command class for reading all events from a stream.
```

Comment:
I just realised that ReadStreamEvents currently inherits "PageStreamEventsBehaviour" which I think is wrong. I think maybe we can inline the _fetch_page_message method and remove the PageAllEventsBehaviour.

The Behaviour classes only exist because I need to min-in some behaviour across multiple conversations, but I'm not sure that's true here.